src/components/Hero/Hero.tsx

```diff
@@ -48,84 +40,137 @@ export default function Hero(props: React.PropsWithChildren<HeroProps>) {
+  }
+
+  if (heroType === HeroTypes.Primary && image) {
+    console.warn(
+      `Warning: the "image" prop has been passed, but PRIMARY hero will not use it`
+    );
+    modifiers.push("warning");
   }
 
-  if (backgroundImageSrc && image) {
-    throw new Error(
-      `Please only either backgroundImageSrc or image into Hero, got both`
+  if (heroType === HeroTypes.Secondary && backgroundImageSrc) {
+    console.warn(
+      `Warning: the "backgroundImageSrc" prop has been passed, but SECONDARY hero will not use it`
     );
+    modifiers.push("warning");
+  }
+
+  if (heroType === HeroTypes.Tertiary && (backgroundImageSrc || image)) {
+    console.warn(`Warning: TERTIARY hero will not use any of the image props`);
+    modifiers.push("warning");
+  }
+
+  if (heroType === HeroTypes.FiftyFifty && backgroundImageSrc) {
+    console.warn(
+      `Warning: the "backgroundImageSrc" prop has been passed, but FIFTYFIFTY hero will not use it`
+    );
+    modifiers.push("warning");
+  }
+
+  if (heroType === HeroTypes.Campaign && (!backgroundImageSrc || !image)) {
+    console.warn(
+      `Warning: it is recommended to use both "backgroundImageSrc" and "image" props for CAMPAIGN hero`
+    );
+    modifiers.push("warning");
   }
 
   if (heroType !== HeroTypes.Primary && locationDetails) {
-    throw new Error(`Please provide locationDetails only to PRIMARY heroTypes`);
+    console.warn(
+      `Warning: Please provide "locationDetails" only to PRIMARY hero`
+    );
+    modifiers.push("warning");
   }
 
-  const backgroundImageStyle = backgroundImageSrc
-    ? { backgroundImage: "url(" + backgroundImageSrc + ")" }
-    : { backgroundColor: backgroundColor };
+  let backgroundImageStyle = {};
+  if (heroType === HeroTypes.Primary) {
+    backgroundImageStyle = backgroundImageSrc
+      ? { backgroundImage: "url(" + backgroundImageSrc + ")" }
+      : null;
+  } else if (
+    heroType !== HeroTypes.Secondary &&
+    heroType !== HeroTypes.FiftyFifty
+  ) {
+    backgroundImageStyle = backgroundImageSrc
+      ? { backgroundImage: "url(" + backgroundImageSrc + ")" }
+      : { backgroundColor: backgroundColor };
+  }
 
   let contentBoxStyling = {};
   if (
     heroType === HeroTypes.Primary ||
     heroType === HeroTypes.Tertiary ||
-    heroType === HeroTypes.FiftyFifty
+    heroType === HeroTypes.FiftyFifty ||
+    heroType === HeroTypes.Campaign
   ) {
-    if (foregroundColor && backgroundColor) {
-      contentBoxStyling = {
-        color: foregroundColor,
-        backgroundColor: backgroundColor,
-      };
-    } else if (foregroundColor || backgroundColor) {
-      const receivedColor = foregroundColor
-        ? "foregroundColor"
-        : "backgroundColor";
-      throw new Error(
-        `Please provide both foregroundColor and backgroundColor to Hero, only got ` +
-          receivedColor
-      );
-    }
+    contentBoxStyling = {
+      color: foregroundColor,
+      backgroundColor: backgroundColor,
```

Comment:
Not necessary but you can do the javascript shorthand for objects where the key and value names are the same. So you can do
```
contentBoxStyling = {
  color = foregroundColor,
  backgroundColor,
}
```

Reply:
Cool.  Did not know that.

Reply:
Done.